ui_framework/components/icon/delete_icon.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { KuiIcon } from './kui_icon';
+
+export function DeleteIcon() {
+  return <KuiIcon className="fa-trash"/>;
```

Comment:
Let's talk about this in person.  There's going to be a ton of back and forth here, and we have that opportunity coming up in March.  For now, let's not block this react progress on it.  We can always do a small code shift to fix these new changes in March if that's what we decide.